assets/src/blocks/Timeline/TimelineEditor.js

```diff
@@ -0,0 +1,196 @@
+  {label: 'Dutch', value: 'nl'},
+  {label: 'English', value: 'en'},
+  {label: 'English (24-hour time)', value: 'en-24hr'},
+  {label: 'Esperanto', value: 'eo'},
+  {label: 'Estonian', value: 'et'},
+  {label: 'Faroese', value: 'fo'},
+  {label: 'Farsi', value: 'fa'},
+  {label: 'Finnish', value: 'fi'},
+  {label: 'French', value: 'fr'},
+  {label: 'Frisian', value: 'fy'},
+  {label: 'Galician', value: 'gl'},
+  {label: 'Georgian', value: 'ka'},
+  {label: 'German / Deutsch', value: 'de'},
+  {label: 'Greek', value: 'el'},
+  {label: 'Hebrew', value: 'he'},
+  {label: 'Hindi', value: 'hi'},
+  {label: 'Hungarian', value: 'hu'},
+  {label: 'Icelandic', value: 'is'},
+  {label: 'Indonesian', value: 'id'},
+  {label: 'Irish', value: 'ga'},
+  {label: 'Italian', value: 'it'},
+  {label: 'Japanese', value: 'ja'},
+  {label: 'Korean', value: 'ko'},
+  {label: 'Latvian', value: 'lv'},
+  {label: 'Lithuanian', value: 'lt'},
+  {label: 'Luxembourgish', value: 'lb'},
+  {label: 'Malay', value: 'ms'},
+  {label: 'Myanmar', value: 'my'},
+  {label: 'Nepali', value: 'ne'},
+  {label: 'Norwegian', value: 'no'},
+  {label: 'Polish', value: 'pl'},
+  {label: 'Portuguese', value: 'pt'},
+  {label: 'pt-br', value: 'Portuguese (Brazilian)'},
+  {label: 'Romanian', value: 'ro'},
+  {label: 'Romansh', value: 'rm'},
+  {label: 'Russian', value: 'ru'},
+  {label: 'Serbian - Cyrillic', value: 'sr-cy'},
+  {label: 'Serbian - Latin', value: 'sr'},
+  {label: 'Sinhalese', value: 'si'},
+  {label: 'Slovak', value: 'sk'},
+  {label: 'Slovenian', value: 'sl'},
+  {label: 'Spanish', value: 'es'},
+  {label: 'Swedish', value: 'sv'},
+  {label: 'Tagalog', value: 'tl'},
+  {label: 'Tamil', value: 'ta'},
+  {label: 'Taiwanese', value: 'zh-tw'},
+  {label: 'Telugu', value: 'te'},
+  {label: 'Thai', value: 'th'},
+  {label: 'Turkish', value: 'tr'},
+  {label: 'Ukrainian', value: 'uk'},
+  {label: 'Urdu', value: 'ur'},
+];
+
+const position = [
+	{label: 'Bottom', value: 'bottom'},
+	{label: 'Top', value: 'top'},
+]
+
+let url_desc  = __(
+	'Enter the URL of the Google Sheets spreadsheet containing your timeline data.',
+	'p4ge'
+);
+url_desc += '<br><a href="https://timeline.knightlab.com/#make" target="_blank" rel="noopener noreferrer">';
+url_desc += __(
+	'See the TimelineJS website for a template GSheet.',
+	'p4ge'
+);
+url_desc += '</a><br>';
+url_desc += __(
+	'Copy this, add your own timeline data, and publish to the web.',
+	'p4ge'
+);
+url_desc += '<br>';
+url_desc += __(
+	"Once you have done so, use the URL from your address bar (not the one provided in Google's 'publish to web' dialog).",
+	'p4ge'
+);
+
+const renderEdit = (attributes, toAttribute) => {
+	return (
+    <Fragment>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+					<URLInput
+						label={__('Google Sheets URL', 'p4ge')}
+						placeholder={__('Enter URL', 'p4ge')}
+						help={ <RawHTML>{url_desc}</RawHTML> }
+						value={attributes.google_sheets_url}
```

Comment:
You could destructure `attributes` at the top of the function, or directly in the declaration, rather than repeat `attributes.` all the time 🙂